### ISSUE TITLE
How to do a "git pull"

### ISSUE THREAD

**Issue by refaelsh** [FredrikNoren/ungit#941]
Hello,

I realize I missing something stupid, and I am gonna feel "bad in the morning" for asking this,
but I can't seem to find how I can perform a plain simple `git pull` in the Ungit GUI.

Thanks.

**Comment by ryantheleach** [comment]
Fetch, then drag the remote onto your branch?

**Comment by refaelsh** [comment]
Hmm... Interesting, I did not think about that option. It would be equivalent to `git fetch && git merge` which is, in turn, equivalent to `git pull`. But I am interested in a single GUI button/thingy/clicky/whatever that performs a `git pull`.

**Comment by wonson** [comment]
I support @refaelsh 's idea.

Most of the git client on the market, even git executable, have this.. shortcut?

**Comment by codingtwinky** [comment]
Yes that would work and there is an work around for the pull operation and it would be nice to have a short cut for it.

Should anybody would like to jump on this issue, below is a good starting point.
https://github.com/FredrikNoren/ungit/blob/master/components/graph/git-graph-actions.js

**Comment by wonson** [comment]
Now I think `git pull` is a must.

Sometimes I haven't get update from the branch for a period of time. Using "fetch then checkout" in ungit will cause an error something like "cannot force to checkout the branch", even the local branch is not dirty. But in terminal, a `git pull` fixed all the troubles

**Comment by jpschewe** [comment]
Personally I like the idea that you know what is happening, fetch + merge. Doing pull without any arguments can cause many problems with merges, especially for new users.

**Comment by wonson** [comment]
> fatal: Cannot force update the current branch.

It show this error when I checkout the new remote master frequently (Even I stashed everything), while one line of `git pull` does its job perfectly
